invenio/values.yaml

```diff
@@ -139,6 +146,12 @@ rabbitmq:
   secret_name: "mq-secrets"
   default_password: "mq_password"
   celery_broker_uri: "amqp://guest:mq_password@mq:5672/"
+  persistence:
+    enabled: true
+    name: "rabbitmq-volume"
+    access_mode: ReadWriteMany
+    size: 10G
+    storage_class: ""
```

Comment:
whats the consequence of the empty string as a storage_class? is it an optional param?